stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messageoptions/MessageOptions.kt

```diff
@@ -0,0 +1,68 @@
+package io.getstream.chat.android.compose.ui.components.messageoptions
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material.ripple.rememberRipple
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.compose.state.messages.list.MessageOptionState
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+
+/**
+ * Displays all available selections of [MessageOptionItem] in an internal [Column].
+ *
+ * @param messageOptionStateList List of available options the user can click on.
+ * @param onMessageItemOptionClicked Handler that propagates click events that occur on an individual [MessageOptionItem].
```

Comment:
Same here:

`onMessageItemOptionClicked Handler that propagates click events that occur on an individual [MessageOptionItem].`

->

`Handler that propagates click events on each item.`

Reply:
simplified